src/pint_pal/utils.py

```diff
@@ -1240,12 +1240,10 @@ def check_recentness_noise(tc):
         return None, None
 
     d = os.path.abspath(tc.get_noise_dir())
-    noise_runs = [os.path.dirname(os.path.dirname(os.path.abspath(p))) 
-                  for p in sorted(glob.glob(os.path.join(d,
-                                                    "..",
-                                                    "*.Noise.*",
-                                                    tc.get_source()+"_"+tc.get_toa_type().lower(),
-                                                    "chain*.txt")))]
+    if os.path.isfile(os.path.join(d, "chain*.txt")):
+        noise_runs = glob.glob(os.path.join(d, "chain*.txt"))
+    else:
+        noise_runs = [os.path.dirname(os.path.dirname(os.path.abspath(p))) for p in sorted(glob.glob(os.path.join(d, tc.get_source()+"_"+tc.get_toa_type().lower(), "chain*.txt")))]
```

Comment:
The old version looked for newer chains in all `*.Noise.*` directories. Of course, we don't have those in the new system, but should we have this look up one directory so it can actually find newer noise chains if they exist?